config/environments/development.rb

```diff
@@ -39,4 +39,8 @@
 
   # for WebSockets
   config.middleware.delete Rack::Lock
+
+  config.survivor_club = {
+      websocket_url: 'localhost:3000/websocket'
```

Comment:
Use 2 spaces for indentation in a hash, relative to the start of the line where the left curly brace is.<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.